src/python.cc

```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2020-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright (c) 2020-2021, NVIDIA CORPORATION & AFFILIATES. All rights
```

Comment:
Remove "(c)" from copyright to avoid line wrap. These wrapped lines will fail in the copyright checker